### PR TITLE
Update 01_1946-09-07.qmd

### PR DESCRIPTION
**Pull request by sherry3809**

### CODE REVIEW

File: chapters/01_1946-09-07.qmd [hunk 1]
```diff
@@ -22,11 +22,11 @@ I'm thinking about this now and even for me, the funny comedian on the Jewish st
 
 It's a long way back...but still!
 When you think about the long road you traveled, and you have in mind not only the others, but also yourself, it seems that everything was just yesterday...
-That's why I will start with myself, and as I involve many others as well, and all in all, I believe a piece of history of fifty years of Yiddish theater in America will be reflected in my stories. I am sure that the thousands and thousands Jews people who are interested in the Jewish life of the past and present will find this interesting.
```
Comment (sherry3809) [inline chapters/01_1946-09-07.qmd:25]: I changed jews to jewish

File: chapters/01_1946-09-07.qmd [hunk 2]
```diff
@@ -22,11 +22,11 @@ I'm thinking about this now and even for me, the funny comedian on the Jewish st
...
+That's why I will start with myself, and as I involve many others as well, and all in all, I believe a piece of history of fifty years of Yiddish theater in America will be reflected in my stories. I am sure that the thousands and thousands of Jewish people who are interested in the Jewish life of the past and present will find this interesting.
 
 ---
 
-So, I will start with myself, as is usually the case in a memoir. And right from the beginning I want to say, not unlike many other Jews that came to America from Russia and from many other countries, I did not bring my name to America; My name was not Sam Kestin back home, but {{< var shmuelik-full >} That's the name they gave me. That's what everyone called me, and no one there, God forbid, ever thought about how difficult it was to get such a long name out...
```
Comment (sherry3809) [inline chapters/01_1946-09-07.qmd:29]: so none of this was in view

File: chapters/01_1946-09-07.qmd [hunk 3]
```diff
@@ -22,11 +22,11 @@ I'm thinking about this now and even for me, the funny comedian on the Jewish st
...
+So, I will start with myself, as is usually the case in a memoir. And right from the beginning I want to say, not unlike many other Jews that came to America from Russia and from many other countries, I did not bring my name to America; My name was not Sam Kestin back home, but {{< var shmuelik-full >}}. That's the name they gave me. That's what everyone called me, and no one there, God forbid, ever thought about how difficult it was to get such a long name out...
```
Comment (sherry3809) [inline chapters/01_1946-09-07.qmd:29]: another comment

File: chapters/01_1946-09-07.qmd [hunk 4]
```diff
@@ -22,11 +22,11 @@ I'm thinking about this now and even for me, the funny comedian on the Jewish st
 
 It's a long way back...but still!
 When you think about the long road you traveled, and you have in mind not only the others, but also yourself, it seems that everything was just yesterday...
-That's why I will start with myself, and as I involve many others as well, and all in all, I believe a piece of history of fifty years of Yiddish theater in America will be reflected in my stories. I am sure that the thousands and thousands Jews people who are interested in the Jewish life of the past and present will find this interesting.
+That's why I will start with myself, and as I involve many others as well, and all in all, I believe a piece of history of fifty years of Yiddish theater in America will be reflected in my stories. I am sure that the thousands and thousands of Jewish people who are interested in the Jewish life of the past and present will find this interesting.
```
Comment (sjspielman) [inline chapters/01_1946-09-07.qmd:25]:
```suggestion
That's why I will start with myself, and as I involve many others as well, and all in all, I believe a piece of history of fifty years of Yiddish theater in America will be reflected in my stories. I am sure that the thousands and thousands of Jews who are interested in the Jewish life of the past and present will find this interesting.
```

**Pull request merged**